src/app/cli/src/init/coda_run.ml

```diff
@@ -482,6 +482,53 @@ let setup_local_server ?(client_trustlist = []) ?rest_server_port
                  [%log info]
                    !"Created GraphQL server at: http://localhost:%i/graphql"
                    rest_server_port ) ) ) ;
+  (*Second graphql server with limited queries exopsed*)
+  Option.iter graphql_port_limited ~f:(fun rest_server_port ->
```

Comment:
It seems that much of this could be shared with the original GraphQL initialisation. Can you factor this out as a function?